lua/lspconfig/util.lua

```diff
@@ -99,6 +99,10 @@ end
 M.path = (function()
   local is_windows = uv.os_uname().version:match 'Windows'
 
+  local function escape_wildcards(path)
+    return path:gsub("([%[%]])", "\\%1")
```

Comment:
pattern can be always expanded, this only match `[]` ?

Reply:
correct, I've now added ? and * too.